buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -58,67 +63,165 @@ dependencies {
   testImplementation 'junit:junit:4.13.1'
 }
 
-public class ExampleRunnerTask extends JavaExec {
+class ExampleRunnerTask extends JavaExec {
 
-  @Input
-  public String basePackage = 'com.google.ads.googleads.examples.'
+  // The base package for all examples.
+  private String basePackage = 'com.google.ads.googleads.examples.'
 
+  // The package.ClassName and any arguments required to launch an example,
+  // e.g. "basicoperations.GetCampaigns --customerId 1234567890".
   @Optional
   @Input
   private String exampleArguments
 
-  public ExampleRunnerTask() {
+  ExampleRunnerTask() {
     group = 'Execution'
     description = 'Run a Google Ads API example.'
     errorOutput = System.err
   }
 
+  String getBasePackage() {
+    return this.basePackage
+  }
+
+  void setBasePackage(String basePackage) {
+    this.basePackage = basePackage
+  }
+
+  String getExampleArguments() {
+    return this.exampleArguments
+  }
+
   @Option(option = 'example', description = 'Sets the example to launch and' +
-      ' any arguments. Required for execution. E.g. ' +
-      '"basicoperations.GetCampaigns --customerId 1234567890"')
-  public void setExampleArguments(String exampleArguments) {
+    ' any arguments. Required for execution. E.g. ' +
+    '"basicoperations.GetCampaigns --customerId 1234567890"')
+  void setExampleArguments(String exampleArguments) {
     this.exampleArguments = exampleArguments.trim()
     int firstSpaceIndex = this.exampleArguments.indexOf(' ')
 
     // No additional arguments were passed, just the example name.
     if (firstSpaceIndex == -1) {
-      main = basePackage + this.exampleArguments
+      main = this.basePackage + this.exampleArguments
     }
     // Otherwise, separate the input and set the arguments to pass to the
     // main class.
     else {
-      main = basePackage + this.exampleArguments[0..firstSpaceIndex - 1]
+      main = this.basePackage + this.exampleArguments[0..firstSpaceIndex - 1]
       argsString(exampleArguments[(firstSpaceIndex + 1)..-1])
     }
   }
 
   @TaskAction
   @Override
-  public void exec() {
+  void exec() {
     if (!(exampleArguments?.trim())) {
       throw new GradleException('\033[0;31mMissing example!\033[0m ' +
-          'Please rerun with one provided, e.g. ' +
-          '\033[0;35m--example="basicoperations.GetCampaigns"\033[0m')
+        'Please rerun with one provided, e.g. ' +
```

Comment:
It'd be great if this error message and the one in the subsequent catch block both suggested the `listExamples` task to get the list of examples.

Reply:
Added.